OpenSim/Simulation/MarkersReference.cpp

```diff
@@ -153,4 +148,37 @@ void MarkersReference::setMarkerWeightSet(const Set<MarkerWeight>& markerWeights
     upd_marker_weights() = markerWeights;
 }
 
+void MarkersReference::setDefaultWeight(double weight)
+{
+    set_default_weight(weight); 
+}
+
+void MarkersReference::updateInternalWeights() const
```

Comment:
I see. I would have made `updateInternalWeights()` non-const and done `const_cast<Self*>(this)` from within `getWeights()` to update, but I think you're treating `updateInternalWeights()` like a cache-updating method, which makes sense. Thanks for the explanation.